addons/website_sale/views/templates.xml

```diff
@@ -919,40 +909,58 @@
+                                </t>
+                                <t t-call="website_sale.search">
+                                    <t t-set="search" t-value="False"/>
+                                    <t t-set="_form_classes" t-valuef="mb-2 mb-lg-0"/>
+                                    <t t-set="_classes" t-value="'me-sm-2'"/>
+                                </t>
+
+                            </div>
+                        </div>
                     </div>
                     <div class="row" id="product_detail_main" data-name="Product Page"
                         t-att-data-image_width="website.product_page_image_width"
                         t-att-data-image_layout="website.product_page_image_layout">
-                        <t t-set="image_cols" t-value="website._get_product_page_proportions()"/>
-                        <div t-attf-class="col-lg-#{image_cols[0]} mt-lg-4 o_wsale_product_images position-relative" t-if="website.product_page_image_width != 'none'">
+
+                        <div t-attf-class="o_wsale_product_images position-relative col-xl-#{image_cols[0]} col-lg-#{image_cols[0] + 1} mt-lg-4" t-if="o_product_width != 'none'">
                             <t t-call="website_sale.shop_product_images"/>
                         </div>
-                        <div t-attf-class="col-lg-#{image_cols[1]} mt-md-4" id="product_details">
-                            <t t-set="base_url" t-value="product.get_base_url()"/>
-                            <h1 itemprop="name" t-field="product.name">Product Name</h1>
-                            <span itemprop="url" style="display:none;" t-esc="base_url + product.website_url"/>
-                            <span itemprop="image" style="display:none;" t-esc="base_url + website.image_url(product, 'image_1920')" />
-                            <t t-if="is_view_active('website_sale.product_comment')">
-                                <a href="#o_product_page_reviews" class="o_product_page_reviews_link text-decoration-none">
-                                    <t t-call="portal_rating.rating_widget_stars_static">
-                                        <t t-set="rating_avg" t-value="product.rating_avg"/>
-                                        <t t-set="trans_text_plural">%s reviews</t>
-                                        <t t-set="trans_text_singular">%s review</t>
-                                        <t t-set="rating_count" t-value="(trans_text_plural if product.rating_count > 1 else trans_text_singular) % product.rating_count"/>
+                        <div id="product_details" t-attf-class="{{ 'row m-0 p-0' if o_product_width_fw else 'offset-xl-1 col-lg-' + str(image_cols[1]) }} mt-lg-4">
+                            <div t-attf-class="d-flex">
+                                <div class="flex-grow-1 mb-3">
+                                    <h1 itemprop="name" t-field="product.name">Product Name</h1>
```

Comment:
-- not the exact line --

"100%" layout: I think that share and wishlist buttons should stay in the product's description column (left)

<img width="1165" alt="image" src="https://github.com/odoo/odoo/assets/8558351/1ec16b1b-fcd4-4786-8380-f254363980a6">